### DISCUSSION TITLE
Problems with Puppeteer and Chromedriver, unable to use correct version

### DISCUSSION THREAD

**Discussion by user7699** [webdriverio/webdriverio#11264] [Q&A] [answered]
Hello, having been away for a while I finally tried to make the jump from v.7 -> v.8. I'm having and issue with Puppeteer and Chromedriver which I can't wrap my head around. If I remove package-lock and node_modules and then update to version 8, I'm getting this error:

 ERROR webdriver: Request failed with status 500 due to session not created: session not created: This version of ChromeDriver only supports Chrome version 117
[0-0] Current browser version is 116.0.5845.96 with binary path C:\Users\myusr\.cache\puppeteer\chrome\win64-116.0.5845.96\chrome-win64\chrome.exe

**package.json**
    "@wdio/cli": "^8.0.11",
    "@wdio/local-runner": "^8.0.11",
    "@wdio/mocha-framework": "^8.0.11",
    "@wdio/spec-reporter": "^8.0.11",
    "chromedriver": "^116.0.0",
    "puppeteer": "21.2.1",

**wdio.conf.js**:

  capabilities: [
    {
      browserName: "chrome",
      "goog:chromeOptions": {
        args: [
          "--ignore-certificate-errors",
          "--window-size=1250,500",
          "--headless",
          "--log-level=3",
        ],
        binary: puppeteer.executablePath(),
      },
    },
  ],

I have Chrome v.117 installed in my computer as my daily browser but both Puppeteer and Chromedriver should be v.116 which is stated by the error log.

If I remove both node_modules and package_lock and downgrade back to v.7, everything works as expected and Chrome v.116 used to execute the tests, eg: 

  "@wdio/cli": "^7.27.0",
        "@wdio/local-runner": "^7.27.0",
        "@wdio/mocha-framework": "^7.26.0",
        "@wdio/spec-reporter": "^7.26.0",

I tried to scavange the changelog for any breaking changes but couldn't find any related to this issue. Does anyone know what is causing this issue and how to get around it? I would like to be able to use v.116 which is stated by both puppeteer and chromedriver in package.json and not v.117 which is the current version I'm using for my browser.

**Comment by christian-bromann** [comment, marked as answer]
I advise to let WebdriverIO allow to setup browser for you, read more about this in https://webdriver.io/blog/2023/07/31/driver-management

**Reply by user7699** [reply]
Thank you @christian-bromann, it works like a charm. This was just what I was looking for. This is actually a really nice feature. But I suppose I won't need puppeteer any longer for the sole purpose of specifying a specific chrome version then ...

**Reply by christian-bromann** [reply]
That is correct!
